http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java

```diff
@@ -371,6 +408,20 @@ public interface Builder extends SdkAsyncHttpClient.Builder<NettyNioAsyncHttpCli
          */
         Builder protocol(Protocol protocol);
 
+        /**
+         * If set to {@code ProtocolNegotiation.ALPN}, the request will be made using ALPN, without a fallback protocol.
+         * If ALPN is not supported by the server an exception will be thrown.
+         * <p>
+         * Default value is {@code ProtocolNegotiation.ALPN} for services with H2 protocol setting, with the exception of the
+         * following services: Kinesis, TranscribeStreaming, Lex Runtime v2, Q Business
+         * <p>
+         * Note: For Java 8, ALPN is only supported in versions 1.8.0_251 and newer.
+         * If on unsupported Java version:
+         *  1) Default SDK setting of true -> SDK will fallback to prior knowledge and not use ALPN
```

Comment:
nit: `Default SDK setting of true` and `User explicitly sets value of true` probably need to be updated